Tests/CustomEntitySearchTest/TestData.cs

```diff
@@ -39,7 +41,7 @@ public class TestData
     ]
 }";
         public const string emptyTextWordsNotFoundInput = @"""will you search?""";
-        public const string emptyWordsEmptyEntitiesInput = @"""if you find when searching, i will be sad""";
+        public const string emptyWordsEmptyEntitiesInput = @"""in""";
```

Comment:
I've just pushed a big refactoring of the test project, sorry, the merge will probably be tricky.

Reply:
And you'll probably want to apply the same patterns from the refactored version: I've changed the code and sample data so it doesn't require so many deserializations, and used .NET types instead on serialized strings where possible.